### DISCUSSION TITLE
Is there a way to highlight certain messages based on content?

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [datalust/seq-tickets#2117] [Help]
I need a way to apply distinctive formatting to some events and I find that the severity is too limiting with essentially only 5 levels available and two of those reserved for errors and warnings. I kind of need another level between Info and Warning, something like "Important" so I can then format appropriately. I'm using NLog and [I've asked the NLog team for this](https://github.com/NLog/NLog/issues/5465) and, as you might expect, the answer is "No, the severity levels are cast in stone". So I need a way to format my output in Seq based on an event property other than the severity level; one that I can determine and populate. Is there a way to do this?

**Comment by nblumhardt** [comment]
Hi Tim! Currently the best match for this would probably be to show an additional property as a column (green tick icon - show as column).

To avoid having this appear on events that aren't the ones of interest, you could attach that property conditionally (at NLog) to only the events that need extra disambiguation.

For example, you might call the property `SubLevel` and give it a value `'Important'` for the events in question.

Another slightly more adventurous option :-) would be to fork _NLog.Targets.Seq_ (it's a fairly small project in https://github.com/datalust/nlog-targets-seq) and use some cue client-side to change the outbound `@l` (level) property to a custom value like `'Important'` for the events in question. Then, server-side you can use custom CSS in Settings > Instance to pick a different color for the "important" level dots.

Hope this helps!

**Comment by snakefoot** [comment]
Created https://github.com/datalust/nlog-targets-seq/pull/75 as suggestion for how to override mapping from NLog LogLevel to Seq Level.

**Comment by snakefoot** [comment]
Nuget-package released with ability to override SeqLevel:

- https://www.nuget.org/packages/NLog.Targets.Seq/3.1.1-dev-00154
